packages/mermaid/src/docs/ecosystem/integrations-community.md

```diff
@@ -97,6 +97,8 @@ Content Management Systems/Enterprise Content Management
   - [Plugin for Mermaid.js](https://emersonbottero.github.io/vitepress-plugin-mermaid/)
 - [VuePress](https://vuepress.vuejs.org/)
   - [Plugin for Mermaid.js](https://github.com/eFrane/vuepress-plugin-mermaidjs)
+- [ApostropheCMS](https://apostrophecms.com/)
+  - [Extension for Mermaid.js](https://github.com/BoDonkey/mermaid-extension)
```

Comment:
We're sorting the extensions alphabetically, so please move the line up to it's correct position :)

Reply:
Ah, sorry I missed that!